packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md

```diff
@@ -61,20 +64,52 @@ At a high-level, the following workflow occurs:
 Verify that your org has the Phone authenticator added and enabled for **Authentication, Recovery**.
 
 1. In the Admin Console, go to **Security** > **Authenticators**.
-1. On the **Setup** tab, verify that the Phone authenticator is listed and that **Authentication, Recovery** is enabled. If not already added, click **Add Authenticator** and then click **Add** on the Phone tile.
-1. Select both **Voice call** and **SMS** as how the verification messages are sent.
-1. Select **Authentication and recovery** as how the Phone authenticator is used.
-1. Click **Add**.
+1. On the **Setup** tab, verify that the Phone authenticator is listed and that **Authentication, Recovery** is enabled in the **Used for** column.
+1. Select **Edit** from the **Actions** menu and verify that both **Voice call** and **SMS** are selected as how the verification messages are sent.
+1. Verify that **Authentication and recovery** is selected as how the Phone authenticator is used.
+1. Click **Save** if made any changes.
+
+> **Note:** If the Phone authentication isn't already added, click **Add Authenticator** and then click **Add** on the Phone tile and make sure that you select the options mentioned above and click **Add**.
 
 ## Update an authentication policy
 
-Update the **Okta Dashboard** [preset authentication policy](https://help.okta.com/okta_help.htm?type=oie&id=ext-preset-auth-policies) to include an additional factor.
+In this section, you update a [preset authentication policy](https://help.okta.com/okta_help.htm?type=oie&id=ext-preset-auth-policies) with another rule. Before we make changes to the authentication policy, we first create a test group and add a test user for this use case example.
+
+### Create a group and add a user
+
+1. In the Admin Console, go to **Directory** > **Groups**.
+1. Click **Add Group** , name it **Support**, and then click **Save**.
+1. Select the group that you just created and on the **People** tab, click **Assign People**.
+1. Add a user to the group for testing. Make sure that the user has a working phone number included in their profile.
+1. Click **Save**.
+
+### Add a rule to the policy
+
+Update the **Okta Dashboard** [preset authentication policy](https://help.okta.com/okta_help.htm?type=oie&id=ext-preset-auth-policies) to include an additional factor by adding an additional rule for the Support group that you just created.
```

Comment:
I'd say you can drop the second "additional" .... ie. "... by adding a rule for the .."